src/App.js

```diff
@@ -1,24 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import {
+  BrowserRouter, Routes, Route, Link,
+} from 'react-router-dom';
+import HomePage from './components/bookPages/HomePage';
+import CategoriesPage from './components/bookPages/CategoriesPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+        </ul>
+      </nav>
+
+      <Routes>
+        <Route exact path="/" component={HomePage} />
+        <Route path="/categories" component={CategoriesPage} />
```

Comment:
- [ ] You have done great to ensure that the Navbar contains two links. However, as the project requirements you should ensure that the Home page displays Books list and book form ☺️.

Your design ☺️ 
![image](https://github.com/bohaz/book-list/assets/99394374/a5bfe0b8-dbd0-4208-94ae-21050adacd6e)

Reply:
Thanks!! I appreciate your professional review. 
![DoneCongratsGIF](https://github.com/bohaz/book-list/assets/127757182/5e4580c6-457f-4452-814e-3ab2a9882e89)